src/container/srv_container.c

```diff
@@ -1220,16 +1220,20 @@
 					min_eph = ec_agg->ea_server_ephs[i].eph;
 			}
 
-			D_ASSERTF(min_eph >= ec_agg->ea_current_eph, DF_U64" < "
-				  DF_U64"\n", min_eph, ec_agg->ea_current_eph);
 			if (min_eph == ec_agg->ea_current_eph)
 				continue;
 
-			D_DEBUG(DB_MD, DF_CONT" sync "DF_U64"\n",
-				DP_CONT(svc->cs_pool_uuid,
-					ec_agg->ea_cont_uuid), min_eph);
+			/**
+			 * NB: during extending or reintegration, the new
+			 * server might cause the minum epoch is less than
+			 * ea_current_eph.
+			 */
+			D_DEBUG(DB_MD, DF_CONT" minum "DF_U64" cur "DF_U64"\n",
+			       DP_CONT(svc->cs_pool_uuid, ec_agg->ea_cont_uuid),
```

Comment:
```suggestion
			 * server might cause the minimum epoch is less than
			 * ea_current_eph.
			 */
			D_DEBUG(DB_MD, DF_CONT" minimum "DF_U64" cur "DF_U64"\n",
				DP_CONT(svc->cs_pool_uuid, ec_agg->ea_cont_uuid),
```

Reply:
This PR landed with the warning unresolved so every PR is going to get notified about it now.

Reply:
oh,  I resolved this by PR-4568. Hope to land soon. Sorry.